caikit/runtime/dump_services.py

```diff
@@ -99,12 +101,37 @@ def dump_http_services(output_dir: str):
 
 
 if __name__ == "__main__":
-    assert len(sys.argv) == 2, f"Usage: {sys.argv[0]} <output_dir>"
-    out_dir = sys.argv[1]
+    parser = argparse.ArgumentParser(
+        description="Dump grpc and http services for inference and train"
+    )
+
+    # Add an argument for the output_dir
+    parser.add_argument(
+        "-o",
```

Comment:
To maintain backwards compatibility, I think we should make this a positional argument ([docs](https://docs.python.org/3/library/argparse.html#core-functionality))